### PR TITLE
Implement addPolicies() and removePolicies()

### PR DESCRIPTION
**Pull request by lwjcjmx123**

### CODE REVIEW

**Comment by nodece** [comment]
@lwjcjmx123 Could you add transaction to `addPolicies` and `removePolicies`?

File: src/adapter.ts [hunk 1]
```diff
@@ -153,6 +167,15 @@ export class SequelizeAdapter implements Adapter {
         await this.sequelize.getRepository(CasbinRule).destroy({where});
     }
 
+    /**
+     * removePolicies removes a policyList rule from the storage.
+     */
+    public async removePolicies(sec: string, ptype: string, rules: string[][]) {
+        for (const rule of rules) {
+          this.removePolicy(sec, ptype, rule);
```
Comment (nodece) [inline src/adapter.ts]: Why not use transaction?

File: src/adapter.ts [hunk 2]
```diff
@@ -137,7 +137,21 @@ export class SequelizeAdapter implements Adapter {
...
+     * addPolicies adds a policyList rules to the storage.
+     */
+    public async addPolicies(sec: string, ptype: string, rules: string[][]) {
+        const lines: CasbinRule[] = [];
+        for (const rule of rules) {
+            const line = this.savePolicyLine(ptype, rule);
+            lines.push(line);
+        }
+        await this.sequelize.transaction(async tx => {
+            await CasbinRule.bulkCreate(lines.map(l => l.get({ plain: true })), { transaction: tx });
```
Comment (nodece) [inline src/adapter.ts:149]: I wonder if using `await CasbinRule.bulkCreate(lines, { transaction: tx });`  is working for avoid call `map` to improve performance.

**Comment by nodece** [comment]
@lwjcjmx123 Could you only keep `Implement addPolicies() and removePolicies()` for git history is clean?

**Comment by lwjcjmx123** [comment]
@nodece DONE

**Comment by nodece** [comment]
LGTM

**Pull request merged**